packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -1036,6 +1037,30 @@ module.exports = (
           uid,
         });
 
+        return {};
+      },
+    },
+    {
+      method: 'POST',
+      path: '/emails/reminders/cad',
+      options: {
+        auth: {
+          strategy: 'sessionToken',
+        },
+      },
+      handler: async function (request) {
+        log.begin('Account.CADReminderEmail', request);
+
+        const sessionToken = request.auth.credentials;
```

Comment:
Since this sends an email, we will need to rate limit it. We would need to add an entry [here](https://github.com/mozilla/fxa/blob/c5eb726cb8c90fcffc136381ab56231a34339a99/packages/fxa-customs-server/lib/actions.js#L42), maybe `sendCadReminder`.

Then call it with `await customs.check(request, sessionToken.email, 'sendCadReminder);`

Reply:
Might need clarification here - my understanding is this handler only calls the method to create the reminder request in Redis, and a little further down we limit this to 2 max, so wouldn't this just exit early if multiple requests from the same sessionToken come in?

Reply:
True, since this doesn't actually send the email it is ok not to put and explicit rate limit check here. 

I've updated the logic a little to create a reminder if one doesn't exist in any interval. Since we are using a sorted set in redis, a user can only have on entry in each time interval.